index.js

```diff
@@ -26,6 +26,7 @@ if (!process.env.token) {
 let appName = 'MyApp';
```

Comment:
`config.js` is a late addition (during work on APOD and event announcement). Another PR to fix a lot of things is definitely welcome. I'm not sure it has widespread acceptance: I only introduce it because strings everywhere is my pet peeve. I try to create some form of constants and use them instead.
cc/ @silentworks @oddoye-david